modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type-api/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/constants/DDMFormFieldTypeConstants.java

```diff
@@ -37,6 +37,8 @@
 
 	public static final String GRID = "grid";
 
+	public static final String HELP_TEXT = "help_text";
```

Comment:
@mateusgomes01 Why did you decide to call this field `Help Text`? Why not something like `Character Options`?

Reply:
We were trying to make a generic `show/hide` field that could be reused later. Would a name like `show_hide` be better than `help_text`?

Reply:
> We were trying to make a generic show/hide field that could be reused later.
It makes sense. Thanks!